scripts/workon.bat

```diff
@@ -71,4 +75,8 @@ if exist "%WORKON_HOME%\%VENV%\%VIRTUALENVWRAPPER_PROJECT_FILENAME%" (
     )
 )
 
+if exist "%WORKON_HOME%\%VENV%\Scripts\postactivate.bat" (

```

Comment:
The mother-project implements both local and global post-activate/deactivate hooks. This code implements the "local" post-activate/deactivate hooks.  #64 implements the global postactivate hook (only).  I would prefer a more comprehensive hook implementation PR...

Reply:
I've added the global versions of these hooks, following #64 but including a global post-deactivate